src/open_inwoner/components/templates/components/File/File.html

```diff
@@ -1,4 +1,4 @@
-{% load i18n link_tags utils icon_tags form_tags dropdown_tags button_tags string_tags %}
+{% load i18n l10n link_tags utils icon_tags form_tags dropdown_tags button_tags string_tags %}
 
 <aside class="file" aria-label="{% trans "Bestand" %} {{ name }}">
     <div class="file__file">
```

Comment:
A flexbox element cannot _both_ be stretched AND centered so you need to wrap it in another flex element, could be a paragraph (but can also be something else) so you could add `<p class="file__symbol">` here.